jgrapht-core/src/main/java/org/jgrapht/alg/scoring/EigenvectorCentrality.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * (C) Copyright 2020, by Sebastiano Vigna.
```

Comment:
We use the following format across the library

(C) Copyright startyear-endyear, by MainAuthorName and Contributors.